public/src/components/channelManagement/tickerEditor.tsx

```diff
@@ -170,7 +158,7 @@ const TickerEditor: React.FC<TickerEditorProps> = ({
             fullWidth
           />
 
-          {tickerSettings.countType === 'money' && (
+          {(tickerSettings.name === 'US' || tickerSettings.name === 'AU') && (
```

Comment:
👍 this is good. If in the future we want to count supporters for US or AU then we can adjust this logic